generators/server/templates/src/test/java/package/CassandraKeyspaceIT.java.ejs

```diff
@@ -42,8 +42,8 @@ public class CassandraKeyspaceIT extends AbstractCassandraTest {
     private Session session;
 
     @Test
-    public void shouldListCassandraUnitKeyspace() throws Exception {
+    public void shouldListCassandraUnitKeyspace() {
```

Comment:
Rename to shouldListCassandraTestKeyspace() ?

Reply:
Oh yes !